Content/Tools/Developer/APITool.cs

```diff
@@ -0,0 +1,50 @@
+﻿using DragonLens.Core.Systems.ThemeSystem;
+using DragonLens.Core.Systems.ToolSystem;
+using Microsoft.Xna.Framework;
+using Microsoft.Xna.Framework.Graphics;
+using Microsoft.Xna.Framework.Input;
+using ReLogic.Content;
+using System;
+using Terraria;
+using Terraria.Localization;
+using Terraria.ModLoader;
+
+internal sealed class APITool : Tool
```

Comment:
Having a unique subclass for 'API' loaded tools seems a bit redundant to me, I'd prefer to get the base class up to snuff rather than having an extra subclass for this that seems to do very little on its own